src/Symfony/Component/Mailer/Bridge/OhMySmtp/Transport/OhMySmtpSmtpTransport.php

```diff
@@ -20,8 +20,12 @@
 use Symfony\Component\Mime\Message;
 use Symfony\Component\Mime\RawMessage;
 
+trigger_deprecation('symfony/oh-my-smtp-mailer', '6.2', 'The "%s" class is deprecated.', OhMySmtpSmtpTransport::class);
```

Comment:
Please mention the new Bridge as replacement here: "…deprecated, use Symfony\Component\Mailer\Bridge\MailPace\Transport\MailPaceSmtpTransport instead."